Index: src/components/nav/Nav.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import Link from \"next/link\";\nimport \"./navbar.css\"\n\nexport default function Nav(){\n    return(\n        <header style={{maxWidth: \"1200px\", width: \"100%\"}}>\n            <nav style={{display: \"flex\", justifyContent: \"space-between\"}}>\n                <Link href=\"/\">GastroGO</Link>\n                \n                <Link href=\"/dashboard\">Dashboard</Link>\n            </nav>\n        </header>\n    )\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/nav/Nav.tsx b/src/components/nav/Nav.tsx
--- a/src/components/nav/Nav.tsx	(revision ab6863c1ad07a5c59dd506bf2c6e8412a423bdf8)
+++ b/src/components/nav/Nav.tsx	(date 1716273823640)
@@ -4,10 +4,14 @@
 export default function Nav(){
     return(
         <header style={{maxWidth: "1200px", width: "100%"}}>
-            <nav style={{display: "flex", justifyContent: "space-between"}}>
-                <Link href="/">GastroGO</Link>
-                
-                <Link href="/dashboard">Dashboard</Link>
+            <nav>
+                <div className="container">
+                    <a className="navbar-brand" href="#">GastroGO</a>
+                    <div className="navbar-buttons">
+                        <button className="btn btn-primary"><a href="/dashboard">Přihlásit</a></button>
+                        <button className="btn btn-secondary"><a href="/dashboard">Registrovat</a></button>
+                    </div>
+                </div>
             </nav>
         </header>
     )
Index: src/components/nav/navbar.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/nav/navbar.css b/src/components/nav/navbar.css
--- a/src/components/nav/navbar.css	(revision ab6863c1ad07a5c59dd506bf2c6e8412a423bdf8)
+++ b/src/components/nav/navbar.css	(date 1716273794866)
@@ -1,0 +1,33 @@
+nav {
+    background-color: white;
+    padding: 10px 20px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navbar-brand {
+    font-weight: bold;
+}
+
+.navbar-buttons {
+    display: flex;
+    gap: 10px;
+}
+
+.btn {
+    padding: 8px 15px;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+.btn-primary {
+    background-color: #007bff;
+    color: white;
+}
+
+.btn-secondary {
+    background-color: #6c757d;
+    color: white;
+}
\ No newline at end of file
